### ISSUE TITLE
Kwargs are not passed from Entrypoint to methods

### ISSUE THREAD

**Issue by xunto** [smagafurov/fastapi-jsonrpc#52]
Hi! Great library. But I stumbled upon an inconsistency.

Is this expected behavior that:

```python
import fastapi_jsonrpc as jsonrpc

api_v1 = jsonrpc.Entrypoint("/rpc", tags=["jsonrpc"])


@api_v1.method()
async def create_link(data: dict):
    pass
```

doesn't make it so `create_link` receives a tag "jsonrpc"?

Looks like a bug to me.

pip freeze:
```
aiojobs==1.1.0
annotated-types==0.5.0
anyio==3.7.1
async-timeout==4.0.2
click==8.1.6
exceptiongroup==1.1.2
fastapi==0.100.1
fastapi-jsonrpc==2.5.0
h11==0.14.0
httptools==0.6.0
idna==3.4
pydantic==1.10.0
pydantic_core==2.4.0
python-dotenv==1.0.0
PyYAML==6.0.1
sniffio==1.3.0
starlette==0.27.0
typing_extensions==4.7.1
uvicorn==0.23.2
uvloop==0.17.0
watchfiles==0.19.0
websockets==11.0.3
```

**Comment by smagafurov** [comment]
Hi!
Not all "kwargs" of entrypoint are correct to transfer to methods, but transfering specifically "tags" seems correct.
Can you make pull request?

**Comment by spumer** [comment]
Fixed in c9151784506140f37a5ca652a2266ec5e3f80d22
